website/src/documentation/patterns/overview.md

```diff
@@ -38,6 +38,9 @@ Pipeline patterns demonstrate common Beam use cases. Pipeline patterns are based
 **Custom window patterns** - Patterns for windowing functions
 * [Using data to dynamically set session window gaps]({{ site.baseurl }}/documentation/patterns/custom-windows/#using-data-to-dynamically-set-session-window-gaps)
 
+**AI Platform integration patterns** - Patterns for Google AI Platform transforms
```

Comment:
Use the full service name here: "Patterns for using the Google Cloud AI Platform Prediction"

Make sure the the service name is consistent throughout the pattern. I noticed that sometimes it's "Google Cloud AI Platform Prediction service", "Google Cloud AI Platform Prediction", etc.

I'd recommend using the full service name ("Google Cloud AI Platform Prediction") the first time its mentioned. Then, refer to the service with the shorthand "AI Platform Prediction". This way, you can use some shorthand but there's no ambiguity about the name of service.

Reply:
Looks good. One minor comment: there are more Google Cloud AI Platform transforms. This pull request is just about Prediction, more patterns could be added later. So I'd stick with `Patterns for Google Cloud AI Platform transforms`